cg/cli/upload/delivery_report.py

```diff
@@ -126,9 +134,9 @@ def _add_delivery_report_to_hk(delivery_report_file, hk_api: hk.HousekeeperAPI,
 
         return None
 
-    def _update_delivery_report_date(status_api, case_id):
-        family_obj = status_api.family(case_id)
-        analysis_obj = family_obj.analyses[0]
+    def _update_delivery_report_date(status_api: Store, case_id, analysis_date):
```

Comment:
Type hints here as well

Reply:
And docstring

Reply:
I find it a bit tricky to follow what's going on with these private functions within another function. From what I can see this one is called in the end of `delivery_report` CLI function only, but it lives in the middle of the code. Is there any specific reason for that?

Reply:
moved to report_api